llm-monitoring-main.ipynb

```diff
@@ -62,36 +62,21 @@
    "execution_count": null,
```

Comment:
<p>Why no output? go over all the cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-monitoring-and-feedback-loop/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>